src/43F0.c

```diff
@@ -74,9 +74,128 @@ HeapNode* _heap_create(s32* addr, u32 size) {
     }
 }
 
-INCLUDE_ASM(s32, "43F0", _heap_malloc);
+#define _heap_alloc_and_update_id(node) { \
```

Comment:
By convention we tend to write macros in SCREAMING_CAPS. I'm personally fine with this but I suspect @ethteck may not be.

It might be possible to match this as a `static inline` func also, or of course we just copy this code into the two places that use it? I'm unsure if we need a macro for 4 lines that only appear twice 📦

Reply:
Gonna leave this unresolved incase @ethteck has an opinion on whether we should have the macro at all

Reply:
Ethan chose 2 temp vars and remove the macro